### PR TITLE
bugfix 129 case sensitive user name login

### LINKED ISSUE #129: Bug: case sensitive user name login
Reproduce:
Loggning in as "ADMIN" is possible, but the user is not an admin (can't access admin page).

- [ ] Add test

### PR DESCRIPTION
**Pull request by martinjonsson01**
The default collation is case-insensitive, so switch it to a case-sensitive one.

image

Closes #129

### CODE REVIEW

File: src/Infrastructure/Migrations/EFDatabaseContextModelSnapshot.cs [hunk 1]
```diff
@@ -111,7 +111,8 @@ protected override void BuildModel(ModelBuilder modelBuilder)
 
                     b.Property<string>("Name")
                         .IsRequired()
-                        .HasColumnType("nvarchar(max)");
+                        .HasColumnType("nvarchar(max)")
+                        .UseCollation("Finnish_Swedish_CS_AS");
```
Comment (jonathan-lindqvist) [inline src/Infrastructure/Migrations/EFDatabaseContextModelSnapshot.cs:115]: 🤖 🇫🇮

**Review by jonathan-lindqvist — APPROVED** [review]
lgtm

File: src/Infrastructure/Migrations/EFDatabaseContextModelSnapshot.cs (hunk 1 above)
Comment (martinjonsson01) [reply src/Infrastructure/Migrations/EFDatabaseContextModelSnapshot.cs:115]: [🇫🇮](https://www.youtube.com/watch?v=6EcDOj6rHro)

**Review by Matte02 — APPROVED** [review]
👍

**Pull request merged**